source/cba_settings_userconfig/cba_settings.sqf

```diff
@@ -24,6 +24,7 @@ force ace_medical_menu_allow = 1;
 force ace_medical_allowUnconsciousAnimationOnTreatment = true;
 force ace_medical_playerDamageThreshold = 2;
 force ace_medical_enableUnconsciousnessAI = 0;
+force force ace_medical_painEffectType = 1;
```

Comment:
`force force` would remove mission maker’s ability to override this setting

Reply:
That is part of the plan, but in case it seems a bit harsh to go straight to remove the option to choose we can absolutely go with a single `force`.  
None of the players at tonight's session (2017-11-25) mentioned having anything against it when directly asked post-mission, but to appease possible naysayers we can absolutely take  the softer route with a single force:
```suggestion
force ace_medical_painEffectType = 1;
```